README.md

```diff
@@ -41,90 +43,157 @@ Add a `.stylelintrc` file to the root of your project:
 }
 ```
 
-> Setting the `syntax` to `scss` is needed for nesting and interpolation support!
-
-Then you need to actually run `stylelint`.
+> **NOTE:** Setting the `syntax` to `scss` is needed for nesting and interpolation support!
 
-Add a `lint:css` script to your `package.json`. This script will run `stylelint` with a path to all of your files containing `styled-components` code:
+Then you need to run `stylelint`. Add a `lint:css` script to your `package.json` which runs `stylelint` with a glob to all of your styled components:
 
 ```JSON
 {
   "scripts": {
-    "lint:css": "stylelint './components/**/*.js'"
+    "lint:css": "stylelint './src/**/*.js'"
   }
 }
 ```
 
-> **NOTE:** Don't worry about passing in files that don't contain any styled-components code – we take care of that.
+> **NOTE:** The processor ignores javascript files that don't contain any `styled-components`, so don't worry about being too broad as long as you restrict it to javascript (or TypeScript).
 
-Now you can lint your CSS by running this script! 🎉
+Now you can lint your CSS by running the script! 🎉
 
 ```
 npm run lint:css
 ```
 
+#### Webpack
+
+If you want to lint on build, rather than as a separate command, you can use the [`stylelint-custom-processor-loader`](https://github.com/emilgoldsmith/stylelint-custom-processor-loader) for webpack.
+
 ### Processor specific stylelint rules
 
-When using this processor a couple of stylelint rules throw errors that you cannot prevent. Like
-'[no-empty-source](https://stylelint.io/user-guide/rules/no-empty-source)' or
-'[no-missing-end-of-source-newline](https://stylelint.io/user-guide/rules/no-missing-end-of-source-newline)'.
+When using this processor a couple of stylelint rules throw errors that cannot be prevented, like [`no-empty-source`](https://stylelint.io/user-guide/rules/no-empty-source) or [`no-missing-end-of-source-newline`](https://stylelint.io/user-guide/rules/no-missing-end-of-source-newline). There's also a couple unnecessary ones like [`no-vendor-prefix` rules](https://stylelint.io/user-guide/rules/property-no-vendor-prefix). (`styled-components` automatically vendor prefixes your code, so you don't need to do it manually)
```

Comment:
You say there are a couple of 'unnecessary rules' here, don't you mean that they're actually necessary? As in, vendor prefixing is unnecessary and therefore we enforce that stylelint rule in our config due to the nature of Styled Components? I think that sentence could use a bit of rephrasing.

Reply:
Sure, feel free to rephrase!